test_scripts/test_setups.py

```diff
@@ -229,6 +232,10 @@ class TestPhase(Enum):
                    {'files': ['m191120_0043.fits', 'm191120_0044.fits',  'm191120_0045.fits', 'm191120_0046.fits'],
                     '--spec_samp_fact': 2.0, '--spat_samp_fact': 2.0}}
 
+_vet = {'keck_lris_red/multi_600_5000_d560':
+                   {},
```

Comment:
I recommend passing in the list of tests here, something like
`'keck_lris_red/multi_600_5000_d560': {'tests': ['test_slitmask.py', 'test_flexure.py'] }`
This would be passed as a "tests" argument to the init of PypeItVet, which could save it for the run method

Reply:
@badpandabear -- how would I get the list of `vet` tests to run only with `develop` and *after* all the 
other tests are done?